stories/Radio.stories.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Radio } from '../src'; // components you used in the story
+import {
+    StandardRadio,
+    RadioStates,
+    DisabledRadio,
+} from './examples/Radio';
+
+<Meta title="Form/Radio" component={Radio} />
+
+# Radio
+
+Radio Options allow the user to select one or more items. They are used for filtering, selection, and toggles. They are typically used in forms, where the user can select one or more items from a set.
+> If you have a single option, avoid using a Radio and use a [Switch component](?path=/docs/form-switch--standard-switch) instead.
+
+## Table of Contents
+
+In this documentation we talk about the Radio component in depth about its usage and its props.
+
+- [Functionality](#functionality)
+    - [Radio States](#radio-states)
+    - [Disabled Radio](#disabled-radio)
+- [Other Props & Functionalities](#other-props--functionalities)
+
+## Functionality
+
+Functionally Radio has two states - checked and unchecked. Thanks to the `checked` prop, you can control
+the state of the Radio. It also comes with a `name` prop, that can be used to difrentiate between multiple Radio Options.
```

Comment:
Also please review the comment I left above about the `name` and adjust accordingly.